docs-md/tutorials/etl.md

```diff
@@ -0,0 +1,614 @@
+      KSQL_CONNECT_CONFIG_STORAGE_TOPIC: "ksql-connect-configs"
+      KSQL_CONNECT_OFFSET_STORAGE_TOPIC: "ksql-connect-offsets"
+      KSQL_CONNECT_STATUS_STORAGE_TOPIC: "ksql-connect-statuses"
+      KSQL_CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR: 1
+      KSQL_CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR: 1
+      KSQL_CONNECT_STATUS_STORAGE_REPLICATION_FACTOR: 1
+      KSQL_CONNECT_PLUGIN_PATH: "/usr/share/kafka/plugins"
+
+  ksqldb-cli:
+    image: confluentinc/ksqldb-cli:0.8.0
+    container_name: ksqldb-cli
+    depends_on:
+      - broker
+      - ksqldb-server
+    entrypoint: /bin/sh
+    tty: true
+```
+
+There are a couple things to notice here. The Postgres image mounts the custom configuration file that we wrote. Postgres adds these configuration settings into its system-wide configuration. The environment variables we gave it also set up a blank database called `customers` along with a user named `postgres-user` that can access it.
+
+We’ve also set up MongoDB as a replica set named `my-replica-set`. Debezium requires that MongoDB run in this configuration to pick up changes from its oplog. In this case, we’re just running a single node replica set.
+
+Finally, note that the ksqlDB server image mounts the `confluent-hub-components` directory, too. The jar files that we downloaded need to be on the classpath of ksqlDB when the server starts up.
+
+Bring up the entire stack by running:
+
+```
+docker-compose up
+```
+
+### Create the customers table in Postgres
+
+It's pretty common for companies to keep their customer data in a relational database. Let's model that information in a Postgres table. Start by logging into the container:
+
+```
+docker exec -it postgres /bin/bash
+```
+
+Log into Postgres as the user created by default:
+
+```
+psql -U postgres-user customers
+```
+
+Create a table that represents the customers. For simplicity's sake, we'll just model this with three columns: an id, a name, and the age of the person:
+
+```sql
+CREATE TABLE customers (id TEXT PRIMARY KEY, name TEXT, age INT);
+```
+
+Seed the table with some initial data:
+
+```sql
+INSERT INTO customers (id, name, age) VALUES ('5', 'fred', 34);
+INSERT INTO customers (id, name, age) VALUES ('7', 'sue', 25);
+INSERT INTO customers (id, name, age) VALUES ('2', 'bill', 51);
+```
+
+### Configure MongoDB for Debezium
+
+Now that Postgres is setup, let's configure MongoDB. Start by logging into the container:
+
+```
+docker exec -it mongo /bin/bash
+```
+
+Log into the Mongo console using the username specified in the Docker Compose file:
+
+```
+mongo -u $MONGO_INITDB_ROOT_USERNAME -p mongo-pw admin
+```
+
+Because MongoDB has been started as a replica set, it needs to be initiated. Run the following command to kick it off:
+
+```javascript
+rs.initiate()
+```
+
+Now that this node has become the primary in the replica set, we need to configure access so that Debezium can replicate changes remotely. Switch into the `config` database:
```

Comment:
```suggestion
Now that this node has become the primary in the replica set, you need to configure access so that Debezium can replicate changes remotely. Switch into the `config` database:
```